x-pack/plugins/maps/public/classes/layers/tiled_vector_layer/tiled_vector_layer.tsx

```diff
@@ -145,18 +146,18 @@ export class TiledVectorLayer extends VectorLayer {
       }
     }
 
-    startLoading(SOURCE_DATA_REQUEST_ID, requestToken, searchFilters);
+    startLoading(SOURCE_DATA_REQUEST_ID, requestToken, requestMeta);
     try {
-      const prevMeta = prevDataRequest ? prevDataRequest.getMeta() : undefined;
+      const prevMeta: VectorSourceRequestMeta | undefined = prevDataRequest
+        ? (prevDataRequest.getMeta() as VectorSourceRequestMeta)
+        : undefined;
       const prevData = prevDataRequest
         ? (prevDataRequest.getData() as MVTSingleLayerVectorSourceConfig)
         : undefined;
       const urlToken =
-        !prevData || isRefreshOnlyQuery(prevMeta ? prevMeta.query : undefined, searchFilters.query)
-          ? uuid()
-          : prevData.urlToken;
+        !prevData || (prevMeta && prevMeta.isForceRefresh) ? uuid() : prevData.urlToken;
```

Comment:
Why are you using prevMeta.isForceRefresh? Shouldn't you use isForceRefresh instead?